### DISCUSSION TITLE
连接到签名服务器出现错误: 丢包4/4 时延9999ms

### DISCUSSION THREAD

**Discussion by Baymacf** [Mrs4s/go-cqhttp#2361] [Q&A]
![image](https://github.com/Mrs4s/go-cqhttp/assets/128383518/a0d9c97e-30c5-4c6a-bb79-1adacecba424)
重启之后突然报出这个错误，本机的网络是正常的，是不是代理的问题？请高手指点，多谢！

**Comment by rhwong** [comment]
你确定开着签名服务器嘛

**Reply by rhwong** [reply]
我只在这个情况下看到了这个错误提示。#2355

**Reply by rhwong** [reply]
但是看你也没用https啊

**Comment by jianghaodn** [comment]
我使用127.0.0.1的时候也遇到了这个问题，虽然我curl 127.0.0.1：port可以得到success的结果。
然后我将ip换成了服务器的lan口的ip192.168.8.1（我是在软路由上面搭建的），然后成功运行了
